pkg/controller/logstorage/esgateway.go

```diff
@@ -75,6 +75,7 @@ func (r *ReconcileLogStorage) createEsGateway(
 		EsInternalCertSecret:       esInternalCertSecret,
 		ClusterDomain:              r.clusterDomain,
 		EsAdminUserName:            esAdminUserName,
+		Replicas:                   install.ControlPlaneReplicas,
```

Comment:
I thought I had commented on this but don't see that anymore. We already pass the install in to esgateway.Config, so there is no need for an explicit Replicas field here.